.pre-commit-config.yaml

```diff
@@ -23,7 +23,7 @@ repos:
       - id: check-toml
       - id: check-vcs-permalinks
       - id: detect-private-key
-        exclude: ^pkg/k8s/secrets/certs\.go$|^pkg/syncthing/certs\.go$|^cmd/deploy/certs\.go$
```

Comment:
this file has been moved now, but we had the old rule, which makes the linter complain